src/app/login/page.tsx

```
@@ -1,65 +1,102 @@
-import { useEffect } from 'react';
+import { useEffect, useState, useCallback } from 'react';
 import { toast } from 'sonner';
 
 import { ICON_PATHS } from '@/constants/icons';
 import { Button } from '@/shared';
 import { useToastStore } from '@/stores/useToastStore';
 
-const LoginPage = () => {
+type LoginPageProps = object;
+
+const LoginPage = ({}: LoginPageProps) => {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)